frontends/p4/typeChecking/typeChecker.cpp

```diff
@@ -2137,8 +2142,10 @@ TypeInference::actionCall(bool inActionList,
         LOG2("Action parameter " << dbp(p));
         if (it == arguments->end()) {
             params->push_back(p);
-            if ((p->direction != IR::Direction::None) || !inActionList)
-                typeError("%1%: parameter %2% must be bound", actionCall, p);
+            if(findContext<IR::EntriesList>() == nullptr) {// no checking for table entries entries
```

Comment:
well, I thought so too. As I mentioned in the PR comment, there are a couple of places where I had to do things that should not be just to get through type checking. I'll take a look at the handling of default_action.